cloudify_aws/s3/resources/bucket_object.py

```diff
@@ -247,7 +247,17 @@ def create(ctx, iface, resource_config, **_):
     ctx.instance.runtime_properties[BUCKET] = bucket_name
 
     # Actually create the resource
-    iface.create(resource_config)
+    try:
+        iface.create(resource_config)
+    except NonRecoverableError as e:
+        acl = resource_config.pop('ACL', '')
+        if 'AccessControlListNotSupported' not in str(e) \
+            and 'The bucket does not allow ACLs' not in str(e) \
```

Comment:
I don't thinks `and 'The bucket does not allow ACLs' not in str(e)` is necessary.
```suggestion
```